recipes/mplfinance/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mplfinance" %}
+{% set version = "0.12.7a0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3e7a6f80a0386419b184434dfc74a943f55f86baa906c25fe2a8d338d5edce62
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3.6
+  host:
+    - matplotlib
+    - pandas
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib
```

Comment:
Probably want `matplotlib-base` instead of `matplotlib`. 

https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib